service/web/server/docs/openapi.yaml

```diff
@@ -0,0 +1,623 @@
+                  payload:
+                    type: object
+                    additionalProperties:
+                      type: string
+                      description: Key-value pairs describing the current scenario.
+                  status:
+                    type: integer
+                    example: 200
+    put:
+      summary: Set the test scenario
+      tags:
+        - meta
+      description: Set the scenario for the running test. Optional properties may trigger an initialization (see README).
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                electricity_price:
+                  type: string
+                  description: Identifier for electricity price scenario.
+                time_period:
+                  type: string
+                  description: Named time period for scenario (may perform initialization).
+                temperature_uncertainty:
+                  type: string
+                  description: Temperature uncertainty setting.
+                solar_uncertainty:
+                  type: string
+                  description: Solar uncertainty setting.
+                seed:
+                  type: integer
+                  description: Random seed for scenario sampling.
+      responses:
+        '200':
+          description: Successful response indicating the scenario was updated.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Scenario updated successfully."
+                  payload:
+                    type: object
+                    description: The applied scenario values (echo).
+                    additionalProperties:
+                      type: string
+                  status:
+                    type: integer
+                    example: 200
+  /step/{testid}:
+    parameters:
+      - $ref: '#/components/parameters/TestId'
+    get:
+      summary: Get the simulation time step
+      tags:
+        - meta
+      description: Returns the time step used in the simulation.
+      responses:
+        '200':
+          description: Successful response with the time step.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  message:
+                    type: string
+                    example: "Simulation time step retrieved successfully."
+                  payload:
+                    type: number
+                    description: The time step used in the simulation.
+                  status:
+                    type: integer
+                    example: 200
+    put:
+      summary: Set the simulation time step
```

Comment:
This is not the simulation time step. Rather call it the "control time step." Correct here and also elsewhere as-needed.